.github/workflows/persubmit.yml

```diff
@@ -9,18 +9,20 @@ jobs:
     if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip ci') && github.event.sender.login != 'taichi-gardener' }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
-        python: [3.6, 3.8]
-        with_cc: [OFF]
         include:
+          - os: ubuntu-latest
+            python: 3.6
+            with_cc: OFF
           - os: macos-latest
```

Comment:
Does this mean both Github actions and travis are testing OSX/Linux?

Reply:
Yeah both GitHub actions an Travis are testing OS X, only GitHub actions are testing Linux.